week-2/D-predicates/exercise.js

```diff
@@ -7,12 +7,12 @@
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
 function isNegative(number) {
-
+if(number > 0) return false;
```

Comment:
This isn't quite right. If you pass in a positive number then you would not get `true` returned back to you. That is because you are only handling the case where the number is negative.

How can you fix this?